orchestra/admin.py

```diff
@@ -186,6 +187,13 @@ class TodoAdmin(AjaxSelectAdmin):
     list_filter = ('task__project__workflow_version',)
 
 
+@admin.register(TodoQA)
+class TodoQAAdmin(AjaxSelectAdmin):
+    list_display = ('id', 'created_at', 'todo', 'approval_reason', 'approved')
+    ordering = ('-created_at',)
+    search_fields = ('todo_description',)
```

Comment:
todo__description (two underscores)

do you also want approval_reason (one underscore)?